kore/src/Kore/Repl.hs

```diff
@@ -101,6 +106,46 @@ runRepl tools simplifier predicateSimplifier axiomToIdSimplifier axioms' claims'
             , labels  = Map.empty
             }
 
+    addIndexesToAxioms
+        :: [Axiom level]
+        -> [Axiom level]
+    addIndexesToAxioms axs =
+        fmap (Axiom . addIndex) (zip (fmap unAxiom axs) [0..(length axs)])
+
+    addIndexesToClaims
+        :: [Axiom level]
```

Comment:
This is a bit confusing. I would rather just pass `length axioms` and take an `Int` here.